raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -370,14 +370,23 @@ private void onBecomeLeader(long currentTimeMs) {
         );
     }
 
+    private List<Voter> getVotersFromVoterId(Set<Integer> voterIds) {
```

Comment:
nit: we can make this a `static` function and rename it to something like `convertToVoters`.

Reply:
Just wanted to know any particular reason you want this to be a static method?

Reply:
Just a nitpick. It doesn't need to be a object method as it doesn't use any of the instance fields. When reading and maintaining a lot of code it is good to know that a method doesn't use instance fields.